fast64_internal/mk64/__init__.py

```diff
@@ -0,0 +1,144 @@
+import bpy
+from bpy_types import PropertyGroup
```

Comment:
```suggestion
from bpy.types import PropertyGroup
```
weird typo, I'm surprised that even worked for you?

Reply:
I am also surprise I just auto import it without see from where